client/store/api/oodlet.js

```diff
@@ -1,19 +1,27 @@
 import Vue from 'vue';
 
-function save(oodlet, oodler) {
-  if(!oodlet.id) {
-    return Vue.http.post('/oodlet', {
-        quantifiedThingies: oodlet.quantifiedThingies,
-        oodlerId: oodler.oodler.id
-      });
-  }
+function load(oodler) {
```

Comment:
Maybe rename it to `active`, you have a `pending` underneath, which goes to `/pending`.